weldx/asdf/schemas/weldx.bam.de/weldx/core/file-1.0.0.yaml

```diff
@@ -0,0 +1,67 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/weldx/core/file-1.0.0"
+tag: "tag:weldx.bam.de:weldx/core/file-1.0.0"
+
```

Comment:
It would be nice add an example here that will be rendered in the documentation

see:
https://github.com/BAMWelDX/weldx/blob/8e14c0f0fa7290d99a760a2c7d7e4efe2996c900/weldx/asdf/schemas/weldx.bam.de/weldx/groove/iso_9692_1_2013_12/VGroove-1.0.0.yaml#L10-L19

Reply:
Unfortunately, that isn't so easy. During the asdf tests, the code of the example gets actually evaluated and the `to_tree` and `from_tree` are executed. Due to the missing file, that always results in a failure. Any idea how to solve that?